third_party_libraries/kml.js

```diff
@@ -3,7 +3,7 @@
 /***
 * Plugin by shramov
 * http://psha.org.ru/b/leaflet-plugins.html
-* 
+*
 * Modifications to work w/ Sencha and custom parsing
```

Comment:
After I saw this, I had the thought that we should look at the license for this since we're poking it. It turns out that (assuming this is the source, which it seems to be) that we're all good with what we're doing and we don't need any extra notes. http://psha.org.ru/cgit/psha/leaflet-plugins.git/tree/LICENSE. I think it would be worthwhile to add a quick note at the top for our changes as well, however.